src/Native/WebGL.js

```diff
@@ -52,66 +52,72 @@ var _elm_community$webgl$Native_WebGL = function () {
   }
 
   function doSettings(gl, settings) {
+    var disable = [];
```

Comment:
💭 this name isn't very clear. I don't know what it should be based on the name alone. A list of disabled draw modes?